src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -991,6 +991,8 @@ public static SyntaxKind GetKeywordKind(string text)
                     return SyntaxKind.UsingKeyword;
                 case "class":
                     return SyntaxKind.ClassKeyword;
+                case "record":
+                    return SyntaxKind.RecordKeyword;
```

Comment:
We can't make this appear as keyword kind. We likely need to special case it in the visitor.